src/definitions/behaviors/form.js

```diff
@@ -368,7 +368,13 @@ $.fn.form = function(parameters) {
             if (initialValue == null) { initialValue = ''; }
             var currentValue = $el.val();
             if (currentValue == null) { currentValue = ''; }
-
+            // multiple select values are returned as arrays which are never equal, so do string conversion first
+            if(Array.isArray(currentValue)) {
+              currentValue = currentValue.toString();
+            }
+            if(Array.isArray(initialValue)) {
```

Comment:
Same as above for this `if` branch.

Reply:
Yes, done by 57ba588f8ef8ba8c78e5c772f697c009767da86b